force-app/main/default/lwc/newServiceSchedule/newServiceSchedule.js

```diff
@@ -169,6 +190,20 @@ export default class NewServiceSchedule extends LightningElement {
         });
     }
 
+    @wire(getRecord, {
```

Comment:
I know this will make @jjbennett sad, but we don't know the session info on load, so I thought this was the fastest way to get that Unit of Measurement value.